portal/src/app/about/AboutContent.jsx

```diff
@@ -11,19 +12,99 @@ class AboutContent extends React.Component {
+                models implemented in our framework:
+                <br />
+                1) quantitative TF-DNA binding specificity models for individual factors, trained on genomic-context
+                protein-binding microarray (gcPBM) data using a core-stratified support vector regression approach.
+                <br />
+                2) models of differential specificity between paralogous TFs, trained on gcPBM data using weighted
+                least squares regression (WLSR). The WLSR approach allows us to identify genomic sites differentially
+                preferred by paralogous TFs, i.e. sites for which the difference in binding between TFs is larger than
+                the variability observed in replicate experiments.
+            </p>
             <div>
-                iMADS enables searching and creating transcription factor binding predictions and preferences.
+                <p className="AboutContent_paragraph">
+                    SEARCH GENOME tab: The iMADS database contains genome-wide predictions for human genome versions
+                    hg19 and hg38, for eleven TFs and ten TF pairs. These predictions can be visualized in the UCSC
+                    Genome Browser, using the track hubs <a href="http://trackhub.genome.duke.edu/gordanlab/tf-dna-binding-predictions/hub.txt">
+                        trackhub.genome.duke.edu/gordanlab/tf-dna-binding-predictions/hub.txt
+                    </a> and <a href="http://trackhub.genome.duke.edu/gordanlab/tf-dna-preferences/hub.txt">
+                        trackhub.genome.duke.edu/gordanlab/tf-dna-preferences/hub.txt
+                    </a>, or directly in the iMADS
+                    web application, using the graphical interface. Users can explore predictions around transcription
+                    start sites (TSSs) of gene, using preloaded or custom gene lists, as well as custom lists of
+                    genomic coordinates.
+                </p>
             </div>
-            <br />
             <div>
-                <span className="AboutContent_source_code_label">Source Code</span>
+                <p className="AboutContent_paragraph">
+                MAKE PREDICTIONS tab: Users can also provide custom sequences as input to obtain predictions according
+                    to any binding specificity or differential binding specificity (preference) model in our framework.
+                </p>
+            </div>
+            <p className="AboutContent_paragraph">
+                All predictions can be downloaded in tab-delimited and csv formats. All data sources used in the iMADS
+                web application are described in the DATA SOURCES tab.
+            </p>
+            <div>
+                <span className="AboutContent_group_label">TECHNOLOGY</span>
+                <ul>
+                    <li>Database management system: PostgreSQL</li>
```

Comment:
Include versions here and for the other pieces? Also, hyperlink each one of them.